src/Apps/ArtQuiz/Routes/ArtQuizResults.tsx

```diff
@@ -11,7 +11,10 @@ interface ArtQuizResultsProps {
 }
 
 const ArtQuizResults: FC<ArtQuizResultsProps> = ({ me }) => {
-  const hasSavedArtworks = me.quiz.savedArtworks.length > 0
+  const savedQuizArtworksCount = useMemo(() => me.quiz.savedArtworks.length, [
```

Comment:
You don't need to memoize accessing a length

Reply:
i'm always so paranoid about passing un-memoized values into child components. 😦 fixed.

Reply:
:) The length of an array isn't computed on the fly, so it's really no different than accessing any other property on an object.